transport/TransportImpl.cpp

```diff
@@ -1065,7 +1065,8 @@ void TransportImpl::internalIceReceived(Endpoint& endpoint,
             return;
         }
 
-        if (_rtpIceSession->getState() == ice::IceSession::State::CONNECTING)
+        const auto iceState = _rtpIceSession->getState();
+        if (iceState == ice::IceSession::State::CONNECTING || iceState == ice::IceSession::State::READY)
         {
             endpoint.registerListener(source, this);
```

Comment:
What happened was that a packet was inbound on the udp endpoint and fit the user name.  Ice session was already in idle and transport just unregistered. But the packet slipped in and caused registration and packet to go to idle ice session.